Exercise_C.js

```diff
@@ -15,4 +15,35 @@ See README.md if you need help running your code.
 
 for (let i = 1; i <= 30; i++) {
   // CODE HERE
-}
+  if(i % 3 === 0){
+    console.log("Hong")
+  } else if (i % 5 === 0){
+    console.log("Kong")
+  }
+    else if((i % 3 === 0) && (i % 5 === 0)){
+    console.log("Hong Kong")
+  }
+  else{
+    console.log(i)
+  }
+}; 
+
+const myArray = [];
+
+for (let i = 1; i <= 30; i++) {
+  if(i % 3 === 0) {
+    myArray.push("Hong")
+  } else if (i % 5 === 0) {
+    myArray.push("Kong")
+  } else if ((i % 3 === 0)&&(i % 5 === 0)) {
+    myArray.push("Hong Kong")
+  } else {
+    myArray.push(i)
+  }
+  
+  
+};
+console.log(myArray);
+
+
+

```

Comment:
This exercise runs correctly, but you do not need an array at the end as this would in turn make the last result of 30 run this all into an array.